backend/entities/Role.cs

```diff
@@ -6,33 +6,21 @@ namespace Pims.Dal.Entities
     /// <summary>
     /// Role class, provides an entity for the datamodel to manage roles.
     /// </summary>
-    public class Role : BaseEntity
+    public class Role : LookupEntity
```

Comment:
This will result in a `Code` property in the `Role`.  I don't think this is the desired implementation of roles.

Reply:
No, it won't. It is extending LookupEntity not CodeEntity. LookupEntity doesn't have codes.

Reply:
Ah my mistake!